src/test/java/core/basesyntax/strategy/AdditionHandlerTest.java

```diff
@@ -6,49 +6,48 @@
 import core.basesyntax.db.StorageTest;
 import core.basesyntax.dto.FruitDto;
 import core.basesyntax.model.Fruit;
-import org.junit.Before;
+import org.junit.BeforeClass;
 import org.junit.Test;
 
 public class AdditionHandlerTest {
-    @Before
-    public void clearStorage() {
+    private static OperationHandler additionHandler;
+
+    @BeforeClass
+    public static void clearStorage() {
+        additionHandler = new AdditionHandler(StorageTest.storage);
         StorageTest.storage.clear();
     }
```

Comment:
now in all handlers the storage is cleared once before all methods - we should do this before/after each test